src/containers/login/login.less

```diff
@@ -0,0 +1,78 @@
+.body-content-container {
```

Comment:
💭 Could we share some of these styles between the two containers? It looks like a lot of them are the same between the two pages.